template/php/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# start with the official Composer image and name it
+FROM composer:1.7.1 AS composer
+
+# continue with the official PHP image
+FROM php:7.2-alpine
+
+# copy the Composer PHAR from the Composer image into the PHP image
+COPY --from=composer /usr/bin/composer /usr/bin/composer
+
+# Composer requirements
+ARG COMPOSER_AUTH='{}'
+ENV COMPOSER_AUTH=${COMPOSER_AUTH}
+
+# Add git
+RUN apk add --no-cache git
+
+# Add watchdog
+RUN echo "Pulling watchdog binary from Github."
+RUN curl -sSL https://github.com/openfaas/faas/releases/download/0.9.4/fwatchdog > /usr/bin/fwatchdog
+RUN chmod +x /usr/bin/fwatchdog
+
+# create non-root user
+RUN addgroup -S app && adduser -S -g app app
+RUN mkdir -p /home/app
+
+# Core
+WORKDIR /home/app
+
+# Function
+COPY index.php php-extension.sh composer.* ./
+RUN chmod u+x ./php-extension.sh && ./php-extension.sh
```

Comment:
@andrew-s is executing the `php-extension.sh` here necessary, you're running again on line 36 below again.